src/node_contextify.cc

```diff
@@ -357,6 +357,7 @@ class ContextifyContext {
 
   static void WeakCallback(const WeakCallbackInfo<ContextifyContext>& data) {
     ContextifyContext* context = data.GetParameter();
+
```

Comment:
That was a line for a printf to check if the context is GCed :)

Thanks, I removed it.